applications/polkascan-cli.md

```diff
@@ -0,0 +1,93 @@
+# Polka Scan CLI
+
+*  **Team Name:** Polkascan CLI Team
+
+*  **Payment Address:** Ethereum Address: 0xE4545705068dec1f68698d032944561Ec332836c
```

Comment:
I think you'll also need an explanation of preferred payment type...

Reply:
I will be accepting USDT as payment type.